plugins/richcombo/plugin.js

```diff
@@ -399,6 +404,37 @@ CKEDITOR.plugins.add( 'richcombo', {
 					listener.removeListener();
 				} );
 				this._.listeners = [];
+			},
+
+			/**
+			 * Selects richcombo item based on the first matching result from the given filter function.
+			 * Filter function takes an object as an argumnet with `value` and `text` fields. Values of those
+			 * fields match to argument passed in `CKEDITOR.ui.richcombo.add` method.
+			 *
+			 * @since 4.14.0
+			 * @param {Function} callback function should return `true` if found matching element
+			 * @param {*} callback.item object containing value and text fields which are compared by this callback
+			 */
+			select: function( callback ) {
+				if ( CKEDITOR.tools.isEmpty( this._.items ) ) {
+					this.createPanel( this._.editor );
```

Comment:
```suggestion
					return;
```

I could mislead you with the previous comment about `this.createPanel` method - I didn't mean that it should be a part of `select` method. Creating a panel based on the value of `this._.items` property seems very implementation-dependent, so I'm for removing this part of code. However, early return if ` CKEDITOR.tools.isEmpty( this._.items )` seems like a good approach.

What I mean was that it should be developer responsibility to make sure that panel is already created when using `panel.select` method. It's rather edged case when someone is trying to use this method when a panel has not been created yet and panel creation should be the toolbar responsibility. But we may add some docs information about that. WDYT?